src/dbnode/generated-source-files.mk

```diff
@@ -155,6 +157,20 @@ genny-map-storage-index-results: install-m3x-repo
 	# Rename generated map file
 	mv -f $(m3db_package_path)/src/dbnode/storage/index/map_gen.go $(m3db_package_path)/src/dbnode/storage/index/results_map_gen.go
 
+# Map generation rule for storage/index/AggregationResultsMap
+.PHONY: genny-map-storage-index-aggregation-results
+genny-map-storage-index-aggregation-results: install-m3x-repo
+	cd $(m3x_package_path) && make hashmap-gen                \
```

Comment:
s/hashmap-gen/idhashmap-gen

It’s the target for genny maps with ident.ID keys, so then you can get rid of the key_type specifier below.